communication/moderation.md

```diff
@@ -202,58 +182,56 @@ umbrella.
 
 ### Discuss
 
-- [Discuss Guidelines](./discuss-guidelines.md)
-- [Moderators](./moderators.md#discuss.kubernetes.io)
-- [Regional Moderators](./moderators.md#regional-category-moderators)
+-   [Discuss Guidelines][]
+-   [Moderators][]
+-   [Regional Moderators][]
 
 ### Mailing List
 
-- [Mailing List Guidelines](./mailing-list-guidelines.md)
-- [Moderators](./moderators.md#mailing-list)
+-   [Mailing List Guidelines][]
+-   [Moderators][15]
 
 ### Slack
 
-- [Slack Guidelines](./slack-guidelines.md)
-- [Moderators](./moderators.md#slack)
+-   [Slack Guidelines][]
+-   [Moderators][16]
 
 ### YouTube
 
-- Moderators should check the Comments section in the community tab regularly
-  for published comments and the "hold for review" sections to see if comments
-  are being flagged by the system.
-- We do NOT use YouTube comments during our live streams, these are checked as
-  OFF in the settings.
-- [Youtube Guidelines](./youtube/youtube-guidelines.md)
-- [Moderators](./moderators.md#youtube-channel)
+-   Moderators should check the Comments section in the community tab regularly
+    for published comments and the "hold for review" sections to see if
+    comments are being flagged by the system.
+-   We do NOT use YouTube comments during our live streams, these are checked as
+    OFF in the settings.
+-   [Youtube Guidelines][]
+-   [Moderators][17]
 
 ### Zoom
 
-- [Zoom Guidelines](./zoom-guidelines.md)
-- [Moderators](./moderators.md#zoom)
+-   [Zoom Guidelines][]
+-   [Moderators][18]
```

Comment:
This entire block should be updated removing the numbers from the links and be given a friendly name or link in the list directly.